graphstorm-processing/graphstorm_processing/data_transformations/dist_transformations/dist_noop_transformation.py

```diff
@@ -66,14 +66,18 @@ def apply(self, input_df: DataFrame) -> DataFrame:
                     f"for column {self.cols[0]}"
                 )
             return input_df
+        elif isinstance(col_datatype, NumericType):
```

Comment:
Is this for parquet input to keep the same data format?

Reply:
Yes, if the incoming data has a numeric type, we just preserve it.